### DISCUSSION TITLE
How can I get the direct homepage link of my project on Github?

### DISCUSSION THREAD

**Discussion by yasinkarax** [community/community#22490] [New to GitHub]
<p>Hi, I made a basic calculator with html,css and js. I want to create a link that people will directly access to project’s index.html page.<br>
<a href="https://github.com/yasinkarax/basicCalculator" rel="noopener nofollow ugc">github repository</a></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>enable GitHub Pages,</p>
<p>go to the repo’s setting, click Pages, host it at <code>main</code> at <code>root</code>,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/4/04cfb5c717106c6115a075df45c14efd9b2e95ba.png" data-download-href="/uploads/short-url/GyVs3eFLeyE2L6O2AV1O1dJCfU.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071412-d5d00f39-670c-4fe1-86c7-95fad3aeb978.png" alt="image" data-base62-sha1="GyVs3eFLeyE2L6O2AV1O1dJCfU" width="690" height="293" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04cfb5c717106c6115a075df45c14efd9b2e95ba_2_690x293.png, https://aws1.discourse-cdn.com/github/original/3X/0/4/04cfb5c717106c6115a075df45c14efd9b2e95ba.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/4/04cfb5c717106c6115a075df45c14efd9b2e95ba.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04cfb5c717106c6115a075df45c14efd9b2e95ba_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1013×431 26.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yasinkarax** [comment]
<p>“Your site is ready to be published at <a href="https://yasinkarax.github.io/basicCalculator/" class="inline-onebox" rel="noopener nofollow ugc">JavaScript Calculator Project</a>”<br>
i clicked and site not found.</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/3/838103a101c35163d8f7e42fe3dfcb90ee59fc1e.jpeg" data-download-href="/uploads/short-url/iLkXx6wqYGTAzjXWfI6qq1HfYzs.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071413-a28f99f6-3021-456a-b274-9b03acc5df28.jpeg" alt="image" data-base62-sha1="iLkXx6wqYGTAzjXWfI6qq1HfYzs" width="574" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838103a101c35163d8f7e42fe3dfcb90ee59fc1e_2_574x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838103a101c35163d8f7e42fe3dfcb90ee59fc1e_2_861x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838103a101c35163d8f7e42fe3dfcb90ee59fc1e_2_1148x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838103a101c35163d8f7e42fe3dfcb90ee59fc1e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1194×1040 48.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>and now it’s up,</p>

**Comment by yasinkarax** [comment]
<p>ıt is seen different on my browser :slight_smile:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/6/16b771cd3ebbb265f01d9260ce7c0bd0203ec5d9.jpeg" data-download-href="/uploads/short-url/3eXx0M7WWTQoa1zwpkD1VNSRAF3.jpeg?dl=1" title="aaaaaaaa" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47751511/181071416-592391a1-1c05-4d43-978a-874c5cbec46d.jpeg" alt="aaaaaaaa" data-base62-sha1="3eXx0M7WWTQoa1zwpkD1VNSRAF3" width="685" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/6/16b771cd3ebbb265f01d9260ce7c0bd0203ec5d9_2_685x499.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/6/16b771cd3ebbb265f01d9260ce7c0bd0203ec5d9_2_1027x748.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/6/16b771cd3ebbb265f01d9260ce7c0bd0203ec5d9.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/6/16b771cd3ebbb265f01d9260ce7c0bd0203ec5d9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">aaaaaaaa</span><span class="informations">1235×901 36.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>nope, click the slide button to enable,</p>
<p>it’s your design, why you don’t know it ?</p>

**Comment by jfullstackdev** [comment]
<p>this is the original one when you first load it,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/7/17fa10595e2ebfbd4ac275944ca0f5a28e5a4c23.jpeg" data-download-href="/uploads/short-url/3q6JX5iJnWkJRndXRb0OC3wqGhd.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071418-d0d9419c-9ce6-4a9b-9b44-6ae8d6f7e0a5.jpeg" alt="image" data-base62-sha1="3q6JX5iJnWkJRndXRb0OC3wqGhd" width="574" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/17fa10595e2ebfbd4ac275944ca0f5a28e5a4c23_2_574x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/7/17fa10595e2ebfbd4ac275944ca0f5a28e5a4c23_2_861x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/7/17fa10595e2ebfbd4ac275944ca0f5a28e5a4c23_2_1148x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/17fa10595e2ebfbd4ac275944ca0f5a28e5a4c23_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1194×1040 43.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yasinkarax** [comment]
<p>I now how to enable it, i meant the appearance of the buttons. :slight_smile:</p>

**Comment by jfullstackdev** [comment]
<p>your posted image is the live site ?</p>
<p>because mine is the live site, you are using CSS, <code>responsive.css</code> file name, but a full responsive is just like Bootstrap or W3.CSS</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181071423-cabeacbb-1ea1-46be-ade7-ec5c181757c0.png" alt="image" data-base62-sha1="qB4r8SuN4aJiBsBKad9tnQ5r4RC" width="407" height="72"></p>
<p>it can be different when it’s local host or any other machine, whether it’s live site vs testing, and depending on the code</p>

**Comment by yasinkarax** [comment]
<p>Buttons don’t look stuck even on small screen in chrome<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/c/3c3f1c766f922ecc76cfc354e607ff8f4135f93e.jpeg" data-download-href="/uploads/short-url/8AXRwzdYPhiicvtpiFEAatrpuNw.jpeg?dl=1" title="v1" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47751511/181071425-3584c9de-c075-40dd-a150-b8662d324060.jpeg" alt="v1" data-base62-sha1="8AXRwzdYPhiicvtpiFEAatrpuNw" width="347" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/c/3c3f1c766f922ecc76cfc354e607ff8f4135f93e_2_347x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/3/c/3c3f1c766f922ecc76cfc354e607ff8f4135f93e_2_520x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/c/3c3f1c766f922ecc76cfc354e607ff8f4135f93e.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/c/3c3f1c766f922ecc76cfc354e607ff8f4135f93e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">v1</span><span class="informations">627×902 40.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yasinkarax** [comment]
<p>but buttons stick on mobile<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/7/075ffdb393ef0ab762b8ec610844222bbfa2a427.jpeg" data-download-href="/uploads/short-url/13f086odTLWwDdWV05uLI46WyFx.jpeg?dl=1" title="v33" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47751511/181071426-8fc8a037-a399-4a4d-b675-acfdb5f50f8b.jpeg" alt="v33" data-base62-sha1="13f086odTLWwDdWV05uLI46WyFx" width="225" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/7/075ffdb393ef0ab762b8ec610844222bbfa2a427_2_225x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/0/7/075ffdb393ef0ab762b8ec610844222bbfa2a427_2_337x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/7/075ffdb393ef0ab762b8ec610844222bbfa2a427_2_450x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/7/075ffdb393ef0ab762b8ec610844222bbfa2a427_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">v33</span><span class="informations">1080×2400 92.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>yup, hopefully you’re getting that point,</p>
<p>either you want to modify your CSS manually, but there is no assurance it will be the same appearance all the time,</p>
<p>or use a responsive framework, like W3.CSS, it’s even lighter than Bootstrap</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://www.w3schools.com/w3css/defaulT.asp">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181071427-c0cafd75-4733-4de0-a62f-3d4bfe15b9e2.png" class="site-icon" width="48" height="48">

      <a href="https://www.w3schools.com/w3css/defaulT.asp" target="_blank" rel="noopener">w3schools.com</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:436/228;"><img src="https://user-images.githubusercontent.com/47092464/181071429-b4f95cc2-2ff0-4fb2-96bf-f03261a4d85d.png" class="thumbnail" width="436" height="228"></div>

<h3><a href="https://www.w3schools.com/w3css/defaulT.asp" target="_blank" rel="noopener">W3.CSS Home</a></h3>

  <p>W3Schools offers free online tutorials, references and exercises in all the major languages of the web. Covering popular subjects like HTML, CSS, JavaScript, Python, SQL, Java, and many, many more.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by yasinkarax** [comment]
<p>I made the buttons with display flex and the size values are given as a percentage %  so they resize themselves and shouldn’t stick together and in chrome it doesn’t stick on the small screen but on mobile it looks different, I’ll look into it</p>

**Comment by Gauravbhatia1211** [comment]
<p><img src="https://user-images.githubusercontent.com/68150602/181071434-8f315fa8-885b-4a33-a8db-21397e9c8b28.png" alt="This is an image" data-base62-sha1="soPnc5vZPuQzBwYvGYK74RK5EPI" width="1" height="1"></p>
